groups/nts/ntsu/ntsu_socketutil.t.cpp

```diff
@@ -158,8 +158,9 @@ void extractZeroCopyNotifications(bsl::list<ntsa::ZeroCopy>* zerocopy,
          it != notifications.notifications().cend();
          ++it)
     {
-        NTSCFG_TEST_TRUE(it->isZeroCopy());
-        zerocopy->push_back(it->zeroCopy());
+        if (it->isZeroCopy()) {
```

Comment:
This change is not what was intended initially. This function should ensure that only ZC notifications are there. Can you change it back? Probably you added it while debugging a bogus TC

Reply:
Yes. I'll do the same for `extraTimestampNotifications`. Done.